testsupport/user_setup.go

```diff
@@ -108,7 +108,7 @@ var HttpClient = &http.Client{
 func postSignup(t *testing.T, route string, identity authsupport.Identity) {
 	require.NotEmpty(t, route)
 	// Call signup endpoint with a valid token.
-	emailClaim := authsupport.WithEmailClaim(uuid.NewV4().String() + "@email.tld")
+	emailClaim := authsupport.WithEmailClaim(uuid.NewV4().String() + "@acme.com")
```

Comment:
Just out of curiosity - what is the purpose of this change?

Reply:
We can configure certain email domains to not require phone verification - for testing purposes, we've configured both `redhat.com` and `acme.com` as excluded domains (see https://github.com/codeready-toolchain/host-operator/pull/291/files#diff-f9db715595ded24a97b8ca746f684769L7).  This change sets the email domain to one of the configured domains in order for the existing tests to still pass, as they would otherwise fail due to requiring verification.

Reply:
Ok, I missed the connection. Thanks a lot for the explanation.